README.md

```diff
@@ -195,6 +195,72 @@ It must be one of the following strings:
 - `same-origin` only sends cookies for the current domain;
 - `include` always send cookies, even for cross-origin calls.
 
+#### `[RSAA].fetch`
+
+A custom [Fetch](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch) implementation, useful for intercepting the fetch request to customize the response status, modify the response payload or skip the request altogether and provide a cached response instead.
+
+`fetch` must be a function that returns a [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) which resolves with an instance of [Request](https://developer.mozilla.org/en-US/docs/Web/API/Response).
```

Comment:
- This should resolve to Response, yeah?
- Also this only has to conform to the Fetch/Request/Response API, not actually to resolve to a Response object. ie, you can resolve to an object that has `json()`, `statusCode`, etc as this middleware isn't doing any sort of type checking here.

Can I suggest phrasing this as:

> A custom [Fetch](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch) implementation, useful for intercepting the fetch request to customize the response status, modifying the response payload or skipping the request altogether and providing a cached response instead.
>
> If provided, the `fetch` option must be a function that conforms to the [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch). Otherwise, the global `fetch` will be used.

Reply:
Oops, that was a typo. Also good point, it doesn't need to be a Response object.